tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -972,40 +970,34 @@ class MklConvOp : public OpKernel {
     output_tf_shape.AddDim((DST_MD.get_size() / sizeof(Toutput)));
     AllocateOutputSetMklShape(context, kOutputIndex_Dst, output_tensor,
                               output_tf_shape, output_mkl_shape);
+    // TODO(bhavanis): Need to integrate the following Add fusion code with
+    // MKL-DNN v1.x
     if (fuse_add_) {
       const Tensor& add_tensor = MklGetInput(context, kInputIndex_Add);
       MklDnnShape add_mkl_shape;
       GetMklShape(context, kInputIndex_Add, &add_mkl_shape);
 
-      // Check if reorder is needed
+      // Check if need reorder
       if (add_mkl_shape == output_mkl_shape) {
-        DCHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
+        CHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
```

Comment:
We don't allow new `CHECK`s in the code anymore. Please use
```c++
auto result = (*output_tensor)->CopyFrom(add_tensor, output_tf_shape);
DCHECK(result);
```
or something.

Reply:
@penpornk Thanks for the quick review! I have changed `CHECK` to `DCHECK`.